README.md

```diff
@@ -4,41 +4,34 @@ A JUnit rule for starting an elasticsearch server on the local machine.
 ## Sample Usage
```

Comment:
Please add Sonar badges same as the zk-client project:
https://github.com/sahabpardaz/zk-client